pkg/cmd/pipelinerun/delete.go

```diff
@@ -259,6 +259,9 @@ func allPipelineRunNames(cs *cli.Clients, keep, since int, ignoreRunning bool, l
 	if ignoreRunning {
 		var pipelineRunTmps = []v1beta1.PipelineRun{}
 		for _, v := range pipelineRuns.Items {
+			if v.Status.Conditions == nil {
```

Comment:
This is not enough... my pipeline run does have conditions with this content
```
{
    "lastTransitionTime": "2022-05-11T14:35:09Z",
    "message": "PipelineRun \"acceptance-tests-preinstalled-kdwv8i\" was cancelled but had errors trying to cancel TaskRuns and/or Runs: Failed to patch TaskRun `acceptance-tests-preinstalled-kdwv8i-generate-ids-r5hdr` with cancellation: taskruns.tekton.dev \"acceptance-tests-preinstalled-kdwv8i-generate-ids-r5hdr\" not found\nFailed to patch TaskRun `acceptance-tests-preinstalled-kdwv8i-setup-testing-accoun-5m9qh` with cancellation: taskruns.tekton.dev \"acceptance-tests-preinstalled-kdwv8i-setup-testing-accoun-5m9qh\" not found\nFailed to patch TaskRun `acceptance-tests-preinstalled-kdwv8i-clone-cli-git-bk5tm` with cancellation: taskruns.tekton.dev \"acceptance-tests-preinstalled-kdwv8i-clone-cli-git-bk5tm\" not found\nFailed to patch TaskRun `acceptance-tests-preinstalled-kdwv8i-clone-plumbing-git-m9655` with cancellation: taskruns.tekton.dev \"acceptance-tests-preinstalled-kdwv8i-clone-plumbing-git-m9655\" not found\nFailed to patch TaskRun `acceptance-tests-preinstalled-kdwv8i-clone-release-tests--kwhbk` with cancellation: taskruns.tekton.dev \"acceptance-tests-preinstalled-kdwv8i-clone-release-tests--kwhbk\" not found\nFailed to patch TaskRun `acceptance-tests-preinstalled-kdwv8i-clone-tutorial-git-5fz54` with cancellation: taskruns.tekton.dev \"acceptance-tests-preinstalled-kdwv8i-clone-tutorial-git-5fz54\" not found\nFailed to patch TaskRun `acceptance-tests-preinstalled-kdwv8i-generate-build-artif-x49pl` with cancellation: taskruns.tekton.dev \"acceptance-tests-preinstalled-kdwv8i-generate-build-artif-x49pl\" not found",
    "reason": "PipelineRunCouldntCancel",
    "status": "Unknown",
    "type": "Succeeded"
  }
```

Reply:
@ppitonak Do you mean we should not delete a `PipelineRun` with status: "Unknown"